packages/components/src/external-link/index.js

```diff
@@ -35,7 +35,7 @@ export function ExternalLink(
 			ref={ ref }
 		>
 			{ children }
-			<VisuallyHidden as="span">
+			<VisuallyHidden>
```

Comment:
While `<div>` and `<span>` have no semantic meaning, they do have user agent styling differences, so best not change this.

Reply:
Does the user agent styling matters if `<VisuallyHidden>` is always visually hidden?

Reply:
In practice it won't make a difference because the styles for hiding position the element absolutely, so it's removed from the flow, and it doesn't matter if its display type is block or inline. 
But there's no good reason for making this change either, so best keep the changeset to what is strictly necessary 🙂